lab-megan/.gitignore

```diff
@@ -0,0 +1,146 @@
+Skip to content
```

Comment:
Just noticed part of Github somehow got into your `.gitignore`, not a bit deal but its a bit funny.